client/rest-high-level/src/main/java/org/opensearch/client/TimedRequest.java

```diff
@@ -73,9 +73,9 @@ public TimeValue timeout() {
     }
 
     /**
-     * Returns the timeout for the request to be completed on the master node
+     * Returns the timeout for the request to be completed on the clustermanager node
```

Comment:
Question: Is it better to replace `master` in the code comments with `cluster manager` or `cluster-manager` ?

Reply:
Answer: if the usage is "master node" or "master role" in the code comments, I replaced with "cluster_manager node or cluster_manager role".